node/subsystem/src/messages.rs

```diff
@@ -215,6 +206,12 @@ pub enum DisputeCoordinatorMessage {
 		/// The validator index passed alongside each statement should correspond to the index
 		/// of the validator in the set.
 		statements: Vec<(SignedDisputeStatement, ValidatorIndex)>,
+		/// Inform the requester once we finished importing.
+		///
+		/// This is, we either discarded the votes, just record them because we
+		/// casted our vote already or recovered availability for the candidate
+		/// successfully.
+		pending_confirmation: oneshot::Sender<ImportStatementsResult>
```

Comment:
The dispute coordinator accepts all statements on all candidates, even undisputed ones. I implemented this in #3348. So there is some middle ground between "once we've cast our vote or recovered availability".

Reply:
Not one 100% what you mean, but I clarified the comment a bit. Hopefully it makes sense - it is quite late over here already.